fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/plans/RecordQueryStreamingAggregationPlan.java

```diff
@@ -97,6 +98,7 @@ public class RecordQueryStreamingAggregationPlan implements RecordQueryPlanWithC
     private final CorrelationIdentifier aggregateAlias;
     @Nonnull
     private final Value completeResultValue;
+    private final SerializationMode serializationMode;
```

Comment:
Should `serializationMode` be added to the plan `equals` method? I think that would be useful in test asserts to ensure that the plan's serialization mode is correctly encoded

Reply:
makes sense. modified.